target/linux/gemini/image/Makefile

```diff
@@ -152,7 +154,9 @@ endef
 TARGET_DEVICES += dlink_dir-685
 
 define Device/dlink_dns-313
-	DEVICE_TITLE := D-Link DNS-313 1-Bay Network Storage Enclosure
+	DEVICE_VENDOR := D-Link
+	DEVICE_MODEL := DNS-313
+	DEVICE_VARIANT := 1-Bay Network Storage Enclosure
```

Comment:
This belongs to MODEL, it's not that there are multiple variants of that device.